src/content/content.service.ts

```
@@ -251,7 +251,47 @@ export class ContentService {
+			}
+
+			const bppIds = this.extractBppIds(response);
+			console.log('bppIds', bppIds);
+			this.logger.log(`Received responses from ${bppIds.length} BPP(s)}`);
+			const arrayOfObjects = this.transformResponseToObjects(response);
+			const uniqueObjects = this.deduplicateObjects(arrayOfObjects);
+			
+			const insertionResponse =
+                    await this.hasuraService.insertCacheData(uniqueObjects , bppIds);
+
+                // Collect all returned items from the response (flatten the result)
```

ESLint

[error] Replace `················` with `↹↹↹`

(prettier/prettier)